macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -2,9 +2,9 @@
   {% if dbt_version >= '1.2.0' %}
         {# This macro is depricated from dbt_utils version 0.9.0, but still hasn't got an equivalent macro at dbt-core #}
         {# Should be replaced to the equivalent macro once it released #}
-        {{ return(dbt_utils.current_timestamp()) }}
+        {{ return(dbt.current_timestamp()) }}
```

Comment:
We need to use the backcompat macro to maintain the same behaviour as before 🙂 
```suggestion
        {{ return(dbt. current_timestamp_backcompat()) }}
```

Reply:
I trim whitespace.